### PR TITLE
(mui-datatables): fix some incorrect types

### PR DESCRIPTION
**Pull request by AntoineDoubovetzky**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/gregnb/mui-datatables#readme
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/mui-datatables/index.d.ts [hunk 1]
```diff
@@ -105,7 +105,7 @@ export interface MUIDataTableColumnOptions {
     hint?: string;
     customHeadRender?: (columnMeta: MUIDataTableCustomHeadRenderer, updateDirection: (params: any) => any) => string;
     customBodyRender?: (value: any, tableMeta: MUIDataTableMeta, updateValue: (s: any, c: any, p: any) => any) => string | React.ReactNode;
-    setCellProps?: (cellValue: string, rowIndex: number, columnIndex: number) => string;
+    setCellProps?: (cellValue: string, rowIndex: number, columnIndex: number) => object;
```
Comment (Favna) [inline types/mui-datatables/index.d.ts]: [The documentation](https://github.com/gregnb/mui-datatables#column-options) says `setCellProps` returns a string. Can you back this with a code snippet from the library? I based my typings primarily on the documentation.

**Review by favna — CHANGES_REQUESTED** [review]
Requested changes due to diversion from library documentation

File: types/mui-datatables/index.d.ts (hunk 1 above)
Comment (AntoineDoubovetzky) [reply types/mui-datatables/index.d.ts]: I opened a PR to fix it in the documentation as well. It has been merged 10 hours ago :)

Comment (Favna) [reply types/mui-datatables/index.d.ts]: I see it: https://github.com/gregnb/mui-datatables/pull/431. Approving review.

**Review by favna — APPROVED** [review]

**Review by favna — APPROVED** [review]

**Comment by Favna** [comment]
Uh not sure what went wrong there with a double approval, but it should just be approved 🤔

**Pull request merged**